.github/workflows/ci.yaml

```diff
@@ -125,10 +126,8 @@ jobs:
         kubectl get nodes
         kubectl get pods -A
 
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: ${{ env.GO_VERSION }}
+    - name: Run direnv
+      uses: jrobsonchase/direnv-action@0.7
```

Comment:
```suggestion
      uses: jrobsonchase/direnv-action@v0.7
```

Reply:
Whoops, should've copy-pasted :roll_eyes: